docs/major-concepts/design-philosophy.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar-position: 1
+---
+
+# Design philosophy
+
+## Transparency
+
+The *manifest file* is the lifeblood of Impact Framework. It defines all the context for an environmental imapct calculation, defining the architectire of an application, the observation period, the pipeline of calculations and transformations to execute, and the environmental impacts to track. This document can then be *executed* to generate impact values. This gives unparalleled transparency to an environmental audit, all in a standard, easy to read format that anyone can read or re-execute.
+
+
+## Verifiability
+
+An Impact Framework manifest file is powerful because anyoen can re-execute it and verify an organization's impact calculation. You can even experiment by swapping out different plugins. The critical concept is that *everything* you need to calculate an impact is provided in the manifest file and *anyone* can re-run a calculation with the manifest file and the lightweight Impact Framework command line tool.
```

Comment:
anyone

Reply:
```suggestion
An Impact Framework manifest file is powerful because anyone can re-execute it and verify an organization's impact calculation. You can even experiment by swapping out different plugins. The critical concept is that *everything* you need to calculate an impact is provided in the manifest file and *anyone* can re-run a calculation with the manifest file and the lightweight Impact Framework command line tool.
```